res/database/Default/user/scr0llbaer_screenplays_adaptations.xml

```diff
@@ -8059,7 +8059,7 @@ The project, which was cancelled by a major producer, is now free to be realized
 			<review min="20" max="60" />
 			<speed min="20" max="60" />
 			<studio_size value="1" />
-			<availability script="TIME_DAYSPLAYED > 12" />
+			<availability script='${.gt:${.worldtime:"daysplayed"}:18}' />
```

Comment:
Semantische Änderung von 12 auf 18

Reply:
wie kommt's zu diesen "semantischen Änderungen"? War wohl auch das Problem in https://github.com/TVTower/TVTower/issues/1188

Reply:
Denke das basiert darauf, dass ich von einem alten Masterstand startere (vor dem DB-PR). Und dann immer schaute, was ich initial committe und das dann anpasste.

Kurzgefasst ... Meine Inkompetenz ;-)

Reply:
mag sein aber ich check's immer noch nich `:-]` 

An den Stellen wurd nichts geändert.

Allerdings gibt's durchaus einige Einträge in dem File mit mal 12 und mal 18.

Könnte es ein schwerwiegendereres Problem sein, dass er Sachen cachet, die er nicht sollte?

Reply:
Man kopiere einen Codeschnipsel, und fuege das beim naechsten Treffer ein...und passt Zahlen an (oder nicht).

Kurzum..sowas editiere ich haendisch im Texteditor (geany).

Reply:
ach soo, dachte die Änderungen hier sind automatisiert (mehr oder weniger)